### ISSUE TITLE
Changing language in development

### ISSUE THREAD

**Issue by Momar-DIENE** [aegif/CmisSync#583]
How to change ressources files from one **language** to another language resources having at his disposal the file **Properties.resources.resx**?

**Comment by nicolas-raoul** [comment]
Not sure... could you please ask at StackOverflow and post the link to your question here?
Thank you!

**Comment by Momar-DIENE** [comment]
yes I posted on stackoverflow it is available on the following link: https://stackoverflow.com/questions/31267416/changing-ressources-files-language-in-development

**Comment by nicolas-raoul** [comment]
Thanks, upvoted :-)
Don't hesitate to add the details asked by Joseph.
